alpine-rails-nginx/etc/nginx/nginx.conf

```diff
@@ -76,6 +66,7 @@ http {
 
   more_set_headers "X-Content-Type-Options: nosniff";
   more_set_headers "X-Frame-Options: deny";
+  more_set_headers "Content-Security-Policy: frame-ancestors 'none'"
```

Comment:
@dkubb I just found this, here: http://www.html5rocks.com/en/tutorials/security/content-security-policy/
```
By default, directives are wide open. If you don’t set a specific policy for a directive, let’s say font-src, then that directive behaves by default as though you’d specified * as the valid source (e.g. you could load fonts from everywhere, without restriction).

The following directives don’t use default-src as a fallback. Remember that failing to set them is the same as allowing anything.

base-uri
form-action
frame-ancestors
plugin-types
report-uri
sandbox
```